gradle.properties

```diff
@@ -2,7 +2,7 @@ org.gradle.caching=true
 org.gradle.parallel=true
 org.gradle.jvmargs='-Dfile.encoding=UTF-8'
 org.gradle.workers.max=3
-version=2201.0.3-SNAPSHOT
+version=2201.0.3-rc1.6-SNAPSHOT
```

Comment:
We will need to change this after the release.

Reply:
Updated to `2201.0.3-SNAPSHOT` for now